src/containers/smartContractComposer/components/actionListGroup.tsx

```diff
@@ -25,6 +25,8 @@ const ActionListGroup: React.FC<ActionListGroupProps> = ({
       <ContractNumberIndicator>
         {actions.length === 1
           ? t('scc.labels.singleActionAvailable')
+          : actions.length === 0
+          ? 'No write actions found'
```

Comment:
Should we use the `writeContractEmptyState.desc` label here?
```suggestion
          ? t('scc.writeContractEmptyState.desc')
```